aws/resource_aws_default_subnet.go

```diff
@@ -15,11 +15,16 @@ func resourceAwsDefaultSubnet() *schema.Resource {
 	dsubnet.Create = resourceAwsDefaultSubnetCreate
 	dsubnet.Delete = resourceAwsDefaultSubnetDelete
 
-	// vpc_id is a required value for Default Subnets
+	// availability_zone is a required value for Default Subnets
 	dsubnet.Schema["availability_zone"] = &schema.Schema{
 		Type:     schema.TypeString,
 		Required: true,
```

Comment:
Should both of these attributes be optional? e.g.

```
--- FAIL: TestAccAWSDefaultSubnet_basic (3.63s)
    testing.go:538: Step 0 error: config is invalid: aws_default_subnet.foo: "availability_zone_id": required field is not set
```

Reply:
Fixed that and also updated the acceptance tests so that they run in parallel cleanly - There is only one default subnet per AZ for the default VPC in a region so make the tests run in separate AZs.